web_src/js/components/RepoActionView.vue

```diff
@@ -335,6 +386,46 @@ const sfc = {
 
     isDone(status) {
       return ['success', 'skipped', 'failure', 'cancelled'].includes(status);
+    },
+
+    closeDropdown() {
+      if (this.menuVisible) this.menuVisible = false;
+    },
+
+    // show at most one of log duration and timestamp (can be both invisible)
+    toggleTimeDisplay(type) {
+      const toToggleTypes = [];
+      const other = type == 'duration' ? 'stamp' : 'duration';
+      this.timeVisible[`log-time-${type}`] = !this.timeVisible[`log-time-${type}`];
+      toToggleTypes.push(type);
+      if (this.timeVisible[`log-time-${other}`]) {
```

Comment:
Sorry, my demo code has a bug, I think it could be like this (from your old logic):

```suggestion
      if (this.timeVisible[`log-time-${type}`] && this.timeVisible[`log-time-${other}`]) {
```